build.rs

```diff
@@ -5,6 +5,21 @@ use std::path::Path;
 pub fn main() {
     let manifest_path =
         Path::new(env!("CARGO_MANIFEST_DIR")).join("Cargo.toml");
+
+    // Paths for reading fixtures used by tests
+    let fixtures_path = Path::new(env!("CARGO_MANIFEST_DIR")).join("fixtures");
```

Comment:
setup ENV variables we can use in unit tests (and integration tests) in Rust. like so: `env!("FIXTURES_TX")` to read out the path of "fixtures/transaction/"